frontend/src/components/HOC/witCheckLogin.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { useContext } from 'react';
+import { UserInfoContext } from '../../context/UserInfoContext';
+import { useNavigate } from '../../lib/Router';
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+function withCheckLogin(Component: React.ComponentType<any>) {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  return function WithCheckAdmin(props: any) {
```

Comment:
```suggestion
import { ComponentProps, FunctionComponent, useContext } from 'react';
import { UserInfoContext } from '../../context/UserInfoContext';
import { useNavigate } from '../../lib/Router';

function withCheckLogin<T>(Component: FunctionComponent<T>) {
  return function WithCheckAdmin(props: ComponentProps<FunctionComponent<T>>) {
```

리액트에서 제공하는 인터페이스를 사용하면 any 없이도 가능합니다 !! 👯